conformity/fields/email.py

```diff
@@ -40,17 +40,7 @@ class EmailAddress(UnicodeString):
     )
     domain_whitelist = ['localhost']
 
-    def __init__(self, message=None, code=None, whitelist=None):
-        """
-        Construct a new email address field.
-
-        :param message: Unused, and will be removed in version 2.0.0
-        :param code: Unused, and will be removed in version 2.0.0
-        :param whitelist: If specified, an invalid domain part will be permitted if it is in this list
-        :type whitelist: iterable
-        """
-        if whitelist is not None:
-            self.domain_whitelist = set(whitelist) if whitelist else set()
+    whitelist = attr.ib(default=[], type=list)
```

Comment:
This is not the proper fix for this. The constructor needs to be left in place, because the whitelist needs to get converted to a set or set as an empty set. But the constructor should be calling `super`, and it isn't currently, which is probably the source of the problem.